### PR TITLE
Add PostgeSQL Tutorial 01

### PR DESCRIPTION
**Pull request by raclepoulpe**
First tutorial for managed PostgreSQL:
Setup and connect a Strapi app to a managed postgre service

### CODE REVIEW

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+
+Click `Create a database service`{.action} to continue.
+
+Within a few minutes your new database service will be deployed. Messages in the OVHcloud Control Panel will inform you when the database is ready to use.
+
+Your database is ready when cluster status is "Ready", node status is green, and number of users is set to "1 user".
+
+![Green statuses on Database Control Panel](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql08.png){.thumbnail}
+
+### Authorise your IP address
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: "Authorize your IP address"

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+
+> [!warning]
+> For security reasons the default network configuration doesn't allow any incoming connections. 
+> To allow access from your application or development environment to your PostgreSQL database service, you must add your IP addresses to the service whitelist.
+
+#### Add your IPs to the DB authorised list
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: "authorized"

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+
+Following the related documentation on how to [authorise the suitable IP addresses](https://docs.ovh.com/gb/en/publiccloud/databases/mongodb/managing-service/#configuring-authorised-ips), add your IPs to the authorised list.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: "authorize"

Comment (scraly) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: and "authorized"

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 4]
```diff
@@ -0,0 +1,444 @@
...
+
+![Green statuses and authorized IPs setted on Database Control Panel](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql10.png){.thumbnail}
+
+### Test connection to the PostgreSQL DataBase
+
+An easy and quick way to test the connection is using the Postgre Command Line Interface (CLI) [psql](https://docs.postgresql.fr/14/app-psql.html){.external}.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: "A quick and easy way to test the connection is to use the Postgre Command Line Interface (CLI)"

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 5]
```diff
@@ -0,0 +1,444 @@
...
+
+> 💡 Tip : If you are familiar with [Docker](https://www.docker.com/){.external} or [Kubernetes](https://kubernetes.io/){.external}, you can use the psql tool inside a postgres Docker [image](https://hub.docker.com/_/postgres){.external} instead of installing the tool locally.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md:127]: Top ! :)

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 6]
```diff
@@ -0,0 +1,444 @@
...
+The useful parameters are:
+
+- The db Host, the db Port
+    - Get them from the "General Information tab"
+
+- The db Name
+    - Get it from the "Databases" tab, usually "defaultdb"
+
+- The db User
+    - Get it from the "Users" tab, usually "avnadmin"
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md:151]: why "avnadmin"? :)

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 7]
```diff
@@ -0,0 +1,444 @@
...
+![Password reseted](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql11.png){.thumbnail}
+
+Now connect to the database with the following command
+
+```sh
+psql --dbname=defaultdb  --host=postgresql-xxxxxxxx-xxxxxxxxx.database.cloud.ovh.net --port=20184 --username=avnadmin  --password 
+```
+Enter the password and press Enter
+
+<pre class="console"><code>$ psql --dbname=defaultdb  --host=postgresql-xxxxxxxx-xxxxxxxxx.database.cloud.ovh.net --port=20184 --username=avnadmin  --password
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: tu veux pas donner un vrai exemple de commande que tu as tapée, avec au pire remplacer les ID par a12b34... ?

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 8]
```diff
@@ -0,0 +1,444 @@
...
+psql (14.1 (Debian 14.1-1.pgdg110+1))
+SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+Type "help" for help.
+
+defaultdb=> 
+</code></pre>
+
+Setup is done, your Managed PostgreSQL database is fully operational, let's go further and use it with <img src="images/strapi-logo-monogram-purple.png" alt="strapi logo" width="12px"> [Strapi](https://strapi.io/){.external}.
+
+### Before beginning, repare your environment
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: "### Before you begin, prepare your environment"

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 9]
```diff
@@ -0,0 +1,444 @@
...
+
+What you need here is to check your nodejs and yarn versions.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: "What you need here is to check your `nodejs` and `yarn` versions."

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 10]
```diff
@@ -0,0 +1,444 @@
...
+According to the official documention on [github](https://github.com/strapi/strapi){.external}, check that your existent nodejs installation matches with "NodeJS >= 12 <= 16".
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: GitHub

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 11]
```diff
@@ -0,0 +1,444 @@
...
+
+❗ No specific version of yarn is specified, this tutorial was made with the 1.22.17 one.
+You can follow this different tuorials if you need to install some of the tools:
+- [nodejs](https://nodejs.org/en/download/package-manager/){.external}
+- [nvm](https://github.com/nvm-sh/nvm){.external}
+- [yarn](https://classic.yarnpkg.com/lang/en/docs/install/){.external}
+
+### Declare the nodejs version
+
+We want to use the latest LTS version of nodejs, so inside your development environment, use this command :
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: use this command:

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 12]
```diff
@@ -0,0 +1,444 @@
...
+
+### Setup a new instance of Strapi
+
+Strapi offers a simple way to install his product, with yarn.
+Run the following command in a terminal:
+```
+yarn create strapi-app my-strapi
+```
+
+You will be prompted to choose between the Quickstart or Custom method, select Custom and press Enter.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: select `Custom`{.action} and press `Enter`{.action}.

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 13]
```diff
@@ -0,0 +1,444 @@
...
+If you choose the Quickstart one, default parameters will be applied and a server will automaticly start.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: automatically

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 14]
```diff
@@ -0,0 +1,444 @@
...
+[3/4] Linking dependencies...
+[4/4] Building fresh packages...
+success Installed "create-strapi-app@4.0.2" with binaries:
+      - create-strapi-app
+? Choose your installation type 
+  Quickstart (recommended) 
+❯ Custom (manual settings) 
+</code></pre>
+
+On next step, select postgres and press Enter
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: select `postgres`{.action} and press `Enter`{.action}

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 15]
```diff
@@ -0,0 +1,444 @@
...
+
+<pre class="console"><code>
+? Choose your installation type Custom (manual settings)
+? Choose your default database client 
+  sqlite 
+❯ postgres 
+  mysql
+</code></pre>
+
+Then, enter your PostgreSQL database parameters, and select No when prompted for enabling SSL connection (We will review that point later)
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: `No`{.action}

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 16]
```diff
@@ -0,0 +1,444 @@
...
+    at readableAddChunk (node:internal/streams/readable:289:9)
+    at Socket.Readable.push (node:internal/streams/readable:228:10)
+    at TCP.onStreamRead (node:internal/stream_base_commons:199:23)
+error Command failed with exit code 1.
+info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
+</code></pre>
+
+This is a known bug, you can follow the issue here: https://github.com/strapi/strapi/issues/12058.
+
+Whe fix it by manually modifying the database configuration file.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: "We can fix it by manually modifying the database configuration file."

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 17]
```diff
@@ -0,0 +1,444 @@
...
+      database: env('DATABASE_NAME', 'defaultdb'),
+      user: env('DATABASE_USERNAME', 'avnadmin'),
+      password: env('DATABASE_PASSWORD', 'xxxxxxxxxxxxxxxxx'),
+      ssl: env.bool('DATABASE_SSL', false),
+    },
+  },
+});
+</code></pre>
+
+Replace the line
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Replace the line:

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 18]
```diff
@@ -0,0 +1,444 @@
...
+<pre class="console"><code>ssl: env.bool('DATABASE_SSL', false),
+</code></pre>
+
+with this one
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: with this line:

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 19]
```diff
@@ -0,0 +1,444 @@
...
+[2021-12-31 13:26:35.898] http: GET /admin/upload-translation-fr-json.0f4b3839.chunk.js (6 ms) 200
+[2021-12-31 13:26:35.902] http: GET /admin/users-permissions-translation-en-json.3e650c44.chunk.js (7 ms) 200
+[2021-12-31 13:26:35.904] http: GET /admin/users-permissions-translation-fr-json.12f3c930.chunk.js (4 ms) 200
+[2021-12-31 13:26:35.906] http: GET /admin/fontawesome-js.2639263c.chunk.js (1 ms) 200
+[2021-12-31 13:26:35.989] http: GET /admin/cropper-css.b71d1229.chunk.js (3 ms) 200
+[2021-12-31 13:26:36.006] http: GET /admin/init (40 ms) 200
+[2021-12-31 13:26:36.173] http: GET /admin/fde9b1ad0670d29a2516.png (1 ms) 200
+</code></pre>
+
+Congratulations! You have just finished the setup, the server starts end open your browser on the admin login creation page http://localhost:1337/admin/auth/register-admin.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md:431]: Congratulations! You have just finished the setup, the server starts and open your browser on the admin login creation page http://localhost:1337/admin/auth/register-admin.

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 20]
```diff
@@ -0,0 +1,444 @@
...
+
+![Strapi admin login creation page](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql12.png){.thumbnail}
+
+Fill the form fields as requested, and press the Let's start button.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: `Let's start`{.action} button

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 21]
```diff
@@ -0,0 +1,444 @@
...
+
+![Strapi dashboard page](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql13.png){.thumbnail}
+
+You are now using a Strapi connected to a OVHcloud managed PostgreSQL database.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: a Strapi app?

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 22]
```diff
@@ -0,0 +1,444 @@
...
+
+### Controls
+
+#### Add an entry to Strapi
+
+Using the admin interface, click on the Content Manager menu, and add a user like:
+
+![Strapi create user page](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql14.png){.thumbnail}
+
+Then press the Save button.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: `Save`{.action}

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 23]
```diff
@@ -0,0 +1,444 @@
...
+
+#### Check the database
+
+As describe above, connect to the PosgreSQL database with psql command line interface.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: `psql`

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 24]
```diff
@@ -0,0 +1,444 @@
...
+```
+$ psql --dbname=defaultdb  --host=postgresql-xxxxxxxx-xxxxxxxxx.database.cloud.ovh.net --port=20184 --username=avnadmin  --password
+```
+Then find in the up_users table our previous created entry, here the user named "demo":
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: `up_users`

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md [hunk 25]
```diff
@@ -0,0 +1,444 @@
...
+----+----------+-----------------+------------------------
+  1 | demo     | demo@mymail.com | 2021-12-31 13:46:08.04
+(1 row)
+</code></pre>
+
+Congratulations! This tutorial is over.
+
+### Cleaning up
+
+To clean your Strapi, make sure it is closed by pressing CTRL+C in the terminal you used to launch it, then delete your installation folder.
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: `CTRL+C`

**Review by scraly — CHANGES_REQUESTED** [review]

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 1 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: debate is open :)

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 5 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md:127]: Thx :)

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 4 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 6 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md:151]: I left the given default username

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 7 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: C'est pourtant une vraie commande :)
c'est plus esthétique avec des a12b34.. que des xxxxxx.. peut-être ?

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 8 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 9 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 10 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 12 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 13 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 15 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 16 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 17 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 18 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 19 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md:431]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 20 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 21 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 22 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 23 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 24 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 25 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md [hunk 26]
```diff
@@ -274,8 +276,7 @@ You can start by doing:
 Done in 662.54s.
 </code></pre>
 
-You could now start you strapi application with yarn commands as described above, but before that, we need to modify an SSL parameter.<br>
-If you don't, this is what will happen:
+If you prefered choose `No` when prompted for enabling SSL connection, this is what happen:
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md]: If you prefer to choose `No` when prompted to enable SSL connection, this is what happens:

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md [hunk 27]
```diff
@@ -335,6 +336,35 @@ with this one:
 
 Save and exit the file.
 
+Now, if you choosed `Yes` when prompted for enabling SSL connection, get the CA certificate from the OVHcloud Control Panel:
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md]: Now, if you choose `Yes` when prompted to enable SSL connection, get the CA certificate from the OVHcloud Control Panel:

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md [hunk 28]
```diff
@@ -335,6 +336,35 @@ with this one:
...
+
+
+![Download certificate](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql16.png){.thumbnail}
+
+and save the generated file into the `config` folder, just beside the `database.js` file, and give it the name `ca-certificate.crt`
+Now open the `config/database.js`, and edit the `ssl` block as:
```
Comment (scraly) [inline pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md]: Now open the `config/database.js` file and modify the `ssl` block as follows:

**Review by scraly — CHANGES_REQUESTED** [review]

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md (hunk 26 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md]: Thx

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md (hunk 27 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md (hunk 28 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 7 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: Done

File: pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md (hunk 1 above)
Comment (raclepoulpe) [reply pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md]: The debate is closed, "authoriZe" won :)

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution @raclepoulpe

**Pull request merged**